packages/story-editor/src/output/test/page.js

```diff
@@ -1450,6 +1495,38 @@ describe('Page output', () => {
 
       await expect(<PageOutput {...props} />).toBeValidAMPStoryPage();
     });
+
+    it('should produce valid output with background audio with captions', async () => {
+      const props = {
+        id: '123',
+        backgroundColor: { type: 'solid', color: { r: 255, g: 255, b: 255 } },
+        page: {
+          id: '123',
+          backgroundAudio: {
+            src: 'https://example.com/audio.mp3',
+            id: 123,
+            mimeType: 'audio/mpeg',
+          },
+          tracks: [
+            {
+              track: 'https://example.com/track.vtt',
+              trackId: 123,
+              trackName: 'track.vtt',
+              id: 'rersd-fdfd-fdfd-fdfd',
+              srcLang: '',
+              label: '',
+              kind: 'captions',
+            },
+          ],
```

Comment:
With #10350 in mind, what do you think about slightly changing the data format like so:

```js
backgroundAudio: {
  resource: { src, id, mimeType }
  track: [ ... ],
  loop: true/false
}
```

This feels much cleaner

Reply:
I do like this idea, how I do think we should do this in a follow up ticket. It would require a migration of data. I think it opens up a larger conversion. For example, I think we should have a audio resource type considering the conversation around https://github.com/GoogleForCreators/web-stories-wp/issues/10002.

Reply:
It's easier to do a migration _before_ introducing new data like `tracks`, not _afterwards_

A dedicated/full audio resource type is not something I want us to invest time in right now. That can be looked into once there is an actual need.

For this use case here, something simple like `resource: { src, id, mimeType }` should suffice

Reply:
Okay, but the resource will need to be `{ src, id, mimeType, length }` for the looping ticket. There is an augment for title and lengthFormatted, as those are useful.

Reply:
I haven't listed all properties for brevity reasons. We should use whatever fields we already have and move them within `resource`. Including `length` (in the separate PR you are already working on) and title and length sounds reasonable.

Reply:
Should story also be updated?

Reply:
What do you mean? Adding a data migration?